### PR TITLE
test: fix numeric compliance test

### PR DESCRIPTION
**Pull request by HemangChothani**

### CODE REVIEW

**Review by larkee — COMMENTED** [review]
There's a lot of skipping relating to decimal which I'm concerned about. Would the changes included in #33 be relevant here?

**Comment by HemangChothani** [comment]
All the tests passes once https://github.com/googleapis/python-spanner/pull/290 PR merged.

**Comment by vi3k6i5** [comment]
Tests look good to me. LGTM.

**Review by vi3k6i5 — APPROVED** [review]
Test changes look good to me. Please wait for Skylar's approval before merging.

**Review by larkee — COMMENTED** [review]
LGTM other than the nit

File: google/cloud/sqlalchemy_spanner/requirements.py
Comment (larkee) [inline google/cloud/sqlalchemy_spanner/requirements.py:81]: Based on this override reason:
```
SPANNER OVERRIDE:
        Cloud Spanner ignores the `0`s after the decimal point if all
        the digits are `0`.
        Overriding the test to avoid the assertion failure.
```

We should not have this property open as it enables a single test that we have to override.

File: google/cloud/sqlalchemy_spanner/requirements.py
Comment (larkee) [inline google/cloud/sqlalchemy_spanner/requirements.py:73]: This property is not true for Spanner so we should not be setting it for open as the test will fail without overrides.

Comment (HemangChothani) [reply google/cloud/sqlalchemy_spanner/requirements.py:73]: This is supported, but till scale =  9.

Comment (larkee) [reply google/cloud/sqlalchemy_spanner/requirements.py:73]: Right, but one of the example numbers is scale = 12. I worry that having this open suggests that we support arbitrary precision `NUMERIC` which is not true.

From a quick search I found [this](https://github.com/zzzeek/sqlalchemy/blob/master/test/requirements.py#L1213) so I still think we should close this property.

File: test/test_suite.py
Comment (larkee) [inline test/test_suite.py:1029]: We are overriding a number of tests because Spanner does not support `scale>9`. But these tests still uses `Numeric` with `scale=12` or `scale=14`. I think it makes sense to change these `Numeric`s to use `scale=9`. WDYT?

**Review by larkee — APPROVED** [review]

**Pull request merged**
